### PR TITLE
fix 6502 errors + add contributors list

### PR DESCRIPTION
**Pull request by bellesea**

### CODE REVIEW

**Comment by bellesea** [comment]
@jessicard does this look good to you?

specifically, what do you think about the way we include the contributor list + is following explaination clear?

> At memory location `$0010` there is the value `#$01` and at memory location `$0011`there is the value `#$02`.
> `JMP ($0010)`
> _At the end of the example, we are jumping to memory location $0102_

> In this instance, we do 2 things:
> 1. Retrive the data at memory location $0010. However, instead of only retrieving data at memory location $0010, we also want to retrieve data at memory location $0011 (as absolute addresses need 16-bit numbers). Remember the part under [Registers and Ram](#registers-and-ram) where we mentioned using multiple memory locations? So here, we retrieve the number #$01 and #$02 from memory location $0010 and $0011 respectively. As 6502 is a little-endian processor, it stores the least significant byte (lower value) first. This means two byte data (like memory locations) store the least significant byte in the first memory address ($0010) and the most significant byte in the second address ($0011) . For example, the following hexadecimal number #$0201, the 01 is the least significant byte and 02 is the most significant byte. (Using a decimal example, 1030, the 30 which refers to thirty is smaller than the 10 which refers to one thousand.)
> 2. Jump to this retrieved absolute location: `$0201`

thank you so much!

File: guide/table-of-contents.md [hunk 1]
```diff
@@ -14,21 +14,7 @@
...
-- [@jakeboxer](https://github.com/jakeboxer)
-- [@tejasag](https://github.com/tejasag)
-- [@HenryBass](https://github.com/HenryBass)
-- [@japrozs](https://github.com/japrozs)
-- [@sporeball](https://github.com/sporeball)
-- [@sampoder](https://github.com/sampoder)
-- [@christinaasquith](https://github.com/christinaasquith)
-- [@quackduck](https://github.com/quackduck)
-- [@maxwofford](https://github.com/maxwofford)
+   Shoutout to these [contributors](contributors.md)
```
Comment (jessicard) [inline guide/table-of-contents.md:17]:
```suggestion
Shoutout to the [contributors](contributors.md) of this guide!
```

**Review by jessicard — APPROVED** [review]
This looks great! Just one small tweak. The rest of it is 👍

**Comment by bellesea** [comment]
thank you so much @jessicard!!!

**Comment by bellesea** [comment]
fixes #59 #58

**Pull request merged**